client/app/utils/places.js

```diff
@@ -0,0 +1,151 @@
+/*
+ Generic utilities for dealing with Google Maps place objects.
+*/
+
+const toRadians = (degrees) => degrees * (Math.PI / 180); // eslint-disable-line no-magic-numbers
+
+/**
+ * Calculate the distance between two points.
+ *
+ * @param {google.maps.LatLng} pointA - first point
+ * @param {google.maps.LatLng} pointB - second point
+ *
+ * @return {Number} distance between the given points (in km?)
```

Comment:
Yes, in km :) This comes from the unit used in `EARTH_RADIUS` constant below: if we want rods or furlongs we can just change the radius to desired unit.